pcdsdevices/tt.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
I think it's risky to make a module with such a short name. Can this be expanded to `timetool.py`?